### PR TITLE
Fuature/hw 4

### PR DESCRIPTION
**Pull request by YariLAN**

### CODE REVIEW

**Comment by YariLAN** [comment]
The problem with all folders was resolved!

File: Student_School_DO/HW4/EntitiesEF/Book.cs [hunk 1]
```diff
@@ -0,0 +1,70 @@
+﻿namespace EntitiesEF
+{
+    public class Book : IEntity<Book>
```
Comment (SolovyovDmitry) [inline Student_School_DO/HW4/EntitiesEF/Book.cs:3]: I wouldn't imagine that you will create separate models for hw3 and hw4 but nevermind)

**Review by SolovyovDmitry — APPROVED** [review]
GJ

File: Student_School_DO/HW4/EntitiesEF/Book.cs (hunk 1 above)
Comment (YariLAN) [reply Student_School_DO/HW4/EntitiesEF/Book.cs:3]: (

**Pull request merged**